test/stdlib/Inputs/VariadicBool/variadicBool.c

```diff
@@ -0,0 +1,15 @@
+#include "variadicBool.h"
+
+int numberOfTrues(int count, va_list arguments) {  
+  int i, total;
+  total = 0;
+  
+  for(i = 0; i < count; i++) {
+    //passing bool to va_arg causes a warning and bool is promoted to int anyway
```

Comment:
I suggest being a little more formal here, since this is the very thing being tested and the next person may have to convince themselves that it's correct. Here's an excerpt from C11 7.16.1.1p2

> [...]if _type_ is not compatible with the type of the actual next argument **(as promoted according to the default argument promotions),** the behavior is undefined[.]

(emphasis mine)